test/e2e/node/mount_propagation.go

```diff
@@ -191,9 +191,12 @@ var _ = SIGDescribe("Mount propagation", func() {
 		cmd = "pidof kubelet"
 		kubeletPid, err := hostExec.IssueCommandWithResult(ctx, cmd, node)
 		framework.ExpectNoError(err, "Checking kubelet pid")
-		kubeletPid = strings.TrimSuffix(kubeletPid, "\n")
-		gomega.Expect(strings.Count(kubeletPid, " ")).To(gomega.Equal(0), "kubelet should only have a single PID in the system (pidof returned %q)", kubeletPid)
-		enterKubeletMountNS := fmt.Sprintf("nsenter -t %s -m", kubeletPid)
+		// In rare cases in the CI, we are seeing two pids for kubelet.
+		// We did some investigating and the last entry of this list is the one that we want
+		// Higher pids are more likely to be created after the first one
+		kubeletPids := strings.Split(kubeletPid, " ")
+		enterKubeletMountNS := fmt.Sprintf("nsenter -t %s -m", kubeletPids[len(kubeletPids)-1])
+		ginkgo.By(enterKubeletMountNS)
```

Comment:
currently this would just be the pid. Can you write what those numbers are?

Reply:
also: we should write the whole output of the `kubeletPid`, not just the successful one (in case this hack breaks one day)

Reply:
updated.